examples/log.py

```diff
@@ -53,6 +56,12 @@ def main():
         if uniform(0, 1) < 0.05:
             warn("Oof. Something went awry.")
 
+        if istep == 50:
+            logmgr.set_watch_interval(50)
+
+        if istep == 1000:
+            logmgr.set_watch_interval(5)
+
```

Comment:
IMO, this leads to awful user experience for someone trying out the demo. It prints three lines for me and then looks like it has crashed, printing nothing for a very long time.